recipes/default.rb

```diff
@@ -0,0 +1,71 @@
+#
```

Comment:
this seems nitpicky, but makes sense. why not have the cookbook called "bluematador" and the recipe called "agent"?

include 'bluematador::agent'

then have the default recipe include agent as well so that this works:

include 'bluematador'

Reply:
Mostly so we can version things differently and manage the dependencies separately.  With most people using berks, if you put everything in one cookbook then they're all tied together.  The agent is extremely lightweight, and probably should be separate from other cookbooks we might make (like for an on-prem solution? idk)